### PR TITLE
Add docker image build/publish workflow

### PR DESCRIPTION
**Pull request by Arochka**

### CODE REVIEW

File: server/start_server.bat [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@echo off
+
+set CSHARP_MODULE_DISABLE_COLLECTIBLE=true
```
Comment (FabianTerhorst) [inline server/start_server.bat]: why?

Comment (Arochka) [reply server/start_server.bat]: Sometimes some packages produce System.NotSupportedException, this is just to avoid that

Comment (FabianTerhorst) [reply server/start_server.bat]: that packages are outdated then. It should not be used.

Comment (Arochka) [reply server/start_server.bat]: Alright, removed ;)

**Comment by C0kkie** [comment]
We dont need that

**Pull request closed without merging**